test/es-module/test-esm-live-binding.mjs

```diff
@@ -0,0 +1,13 @@
+// Flags: --experimental-modules
+
+import '../common';
+import assert from 'assert';
+
+import fs, { readFile } from 'fs';
+
+const s = Symbol();
+
+fs.readFile = () => s;
```

Comment:
Similarly, this doesn’t have to actually have a function - it just needs something to ===

Reply:
because this implementation wraps functions, i feel like testing return value is worthwhile